graylog2-web-interface/src/components/maps/adapter/MaxmindAdapterFieldSet.tsx

```diff
@@ -46,8 +46,8 @@ const MaxmindAdapterFieldSet = ({
   const isCloud = AppConfig.isCloud();
 
   const pathsForCloud = {
-    IPINFO_STANDARD_LOCATION: '/etc/graylog/server/standard_location.mmdb',
-    IPINFO_ASN: '/etc/graylog/server/asn.mmdb',
+    IPINFO_STANDARD_LOCATION: '/etc/server/standard_location.mmdb',
```

Comment:
These values are just examples, right?

Reply:
These are just examples. Thinking about it, I will revert these file path changes for now and doublecheck if there is a case where we should keep the original paths, because actually exists in this dir.

Reply:
I double checked it and removed the changes from `MaxmindAdapterFieldSet`.